apps/fluent-tester/src/TestComponents/Button/ButtonVariantTestSection.tsx

```diff
@@ -39,6 +40,9 @@ export const ButtonVariantTest: React.FunctionComponent = () => {
       <CompoundButton appearance="subtle" secondaryContent="Compound" style={commonTestStyles.vmargin}>
         Subtle
       </CompoundButton>
+      <FAB icon={iconProps} style={commonTestStyles.vmargin}>
```

Comment:
Seems like for Android we're already showing this in the ButtonSizeTestSection. Do we also want to show it for Android here, or should this be just shown on iOS?

Reply:
For Android it is displayed in both Button Variant section (forked out) and the Button sizes section. On iOS it was initially being shown, have brought back the old behaviour